microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -129,8 +128,8 @@ The following shortcuts are always turned off in WebView2.  An asterisk (`*`) in
 | Select Tab (1 - 8) | `Ctrl`+`(1-8)` |
 | Show Favorites Bar `*` | `Ctrl`+`Shift`+`B` |
 | Help | `F1` |
-| Focus Next Pane `*` | `F6` |
-| Focus Previous Pane `*` | `Shift`+`F6` |
+| Focus Next Pane | `F6` |
```

Comment:
@tofuandeve @jasonstephen15 - Should I add a note that F6 & Shift+F6 is currently only enabled in windowed hosting mode, not yet in visual hosting mode?

I'll look at .md file in VS Code to get a better view of where such note could go.

Reply:
@mikehoffms Yes, this is an important information, we should definitely add a note to explicitly call this out.